packages/@uppy/xhr-upload/src/index.js

```diff
@@ -19,6 +21,10 @@ function buildResponseError (xhr, error) {
     error = Object.assign(new Error('Upload error'), { data: error })
   }
 
+  if (isNetworkError(xhr)) {
+    error = new NetworkError(error)
```

Comment:
for consistent behaviour, would it make sense to pass xhr to `NetworkError` and return early like so:

```js
if (isNetworkError(xhr)) {
    error = new NetworkError(error, xhr)
    return error
}
```

and then change [this field](https://github.com/transloadit/uppy/pull/2110/files#diff-dcd2ff4fc4c4c209bccb7194f0206d8fR6) from `originalRequest` to `request`.  Changing the field will make it consistent with how it's done in `xhr-upload` already.

Returning early will prevent any unintended changes on the NetworkError object which may be added after the `if(...)` block. This will make the `NetworkError` class consistent. 

If we agree with this suggestion, we can do the same for `MiniXhrUpload`

Reply:
Agreed, updated!